app/models/user.rb

```diff
@@ -77,12 +77,22 @@ def confirmed?
     confirmed_at.present?
   end
 
-  def super_admin?
-    has_role?(:super_admin)
+  # Generates all role? methods where role is a role_name from config/covidliste.yml
```

Comment:
Maybe it can be more readable without metaprogramming?

```ruby
def has_role?(role_name)
  roles = Rails.application.config.x.covidliste.fetch("admin_roles")
  roles.include?(role_name.to_s)
end
```

Usage:
```ruby
user.has_role?(:admin)
user.has_role?("admin")
```